pallets/polkadex/src/lib.rs

```diff
@@ -345,22 +345,24 @@ impl<T: Trait> Module<T> {
     }
 
     /// Initializes a new Orderbook and stores it in the Orderbooks
-    fn create_order_book(quote_asset_id: T::AssetId, base_asset_id: T::AssetId, trading_pair_id: &T::Hash) {
+    fn create_order_book(quote_asset_id: T::AssetId, base_asset_id: T::AssetId, trading_pair_id: (T::Hash,T::Hash)) {
         let orderbook = Orderbook::new(base_asset_id, quote_asset_id, trading_pair_id.clone());
         <Orderbooks<T>>::insert(trading_pair_id, orderbook);
         <AsksLevels<T>>::insert(trading_pair_id, Vec::<FixedU128>::new());
         <BidsLevels<T>>::insert(trading_pair_id, Vec::<FixedU128>::new());
     }
 
     /// Creates a TradingPairID from both Asset IDs.
-    fn create_trading_pair_id(quote_asset_id: &u32, base_asset_id: &u32) -> T::Hash {
-        (quote_asset_id, base_asset_id).using_encoded(<T as frame_system::Trait>::Hashing::hash)
+    fn create_trading_pair_id(quote_asset_id: &u32, base_asset_id: &u32) -> (T::Hash, T::Hash) {
```

Comment:
This function will be removed when the custom assets pallet is installed as asset ID is represented as a hash and not as u32.

So trading pair will (a,b) where a < b and a, b are the asset hashes